tools/clang/test/DXC/Passes/PIX/validatorUpgrade.ll

```diff
@@ -0,0 +1,64 @@
+; RUN: %dxopt %s -dxil-annotate-with-virtual-regs -hlsl-dxilemit -S | FileCheck %s
+
+; CHECK: !dx.valver = !{![[VALVER:.*]]}
+; CHECK-NOT: ![[VALVER]] = !{i32 1, i32 4}
```

Comment:
This ensures we don't downgrade the version, but is there already a test to make sure we upgrade the validator version in the appropriate scenario somewhere else?

Reply:
Didn't think of manually editing in an older valver. New test added.